utils/check-missing-logos.ts

```diff
@@ -0,0 +1,37 @@
+const {
+  readQuickstartFile,
+  findMainQuickstartConfigFiles,
+} = require('./helpers');
```

Comment:
Looks like we just have one more import that needs to be updated 😄

Reply:
Lol thanks! They were really flying under my radar. I got it removed